components/brave_wallet_ui/nft/components/nft-details/nft-screen.tsx

```diff
@@ -249,7 +249,7 @@ export const NftScreen = ({ selectedAsset, tokenNetwork }: Props) => {
             <NftMultimedia
               as='img'
               visible={!isFetchingNFTMetadata}
-              src={nftMetadata?.imageURL}
+              src={`chrome://image?url=${encodeURIComponent(nftMetadata?.imageURL || '')}&staticEncode=true9`}
```

Comment:
this is an instance where it may be still breaking NFT images, so I left the extra query param number on to track it easier when rendering.

Make sure to set this back to `true` before merging so we actually get static encoding working as expected